src/aiida_epw/workflows/base.py

```diff
@@ -94,8 +94,9 @@ def define(cls, spec):
             required=True,
             serializer=to_aiida_type,
             help=(
-                "The options dictionary for the calculation."
-                "It must be defined in the top-level as a solution of the conflict between `metadata_calculation` of the WorkChain and `metadata` of the Calculation."
+                "Dictionary containing the options for the calculation job."
+                "This input is defined at the top-level to resolve the conflict between"
```

Comment:
I missed this in a previous review, but I don't think the user needs to know about this technicality.